src/android/jni/utils.cpp

```diff
@@ -0,0 +1,173 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include <jni.h>
+#include "error.h"
+#include "../../../include/librealsense2/rs.hpp"
+#include "../../../include/librealsense2/rsutil.h"
+
+
+extern "C" JNIEXPORT void JNICALL
+Java_com_intel_realsense_librealsense_Utils_nProjectPointToPixel(JNIEnv *env, jclass type,
+                                                              jobject pixel_2D, jobject intrinsic,
+                                                              jobject point_3D) {
+    // retrieving float[3] from point_3D
+    jclass point_3D_class = env->GetObjectClass(point_3D);
+    jfieldID point_x_field = env->GetFieldID(point_3D_class, "mX", "F");
+    jfieldID point_y_field = env->GetFieldID(point_3D_class, "mY", "F");
+    jfieldID point_z_field = env->GetFieldID(point_3D_class, "mZ", "F");
+    float* point = new float[3];
```

Comment:
This will leak. Why not float data[3] for temporal ?

Reply:
done